db/migrate/20170313203319_add_assets_details_ref_to_physical_server.rb

```diff
@@ -0,0 +1,5 @@
+class AddAssetsDetailsRefToPhysicalServer < ActiveRecord::Migration[5.0]
+  def change
+    add_reference :physical_servers, :asset_details, index: true, foreign_key: true
```

Comment:
Two things here:

1) Remove the `foreign_key` constraint
2) add `type: :bigint`

Also, @Fryguy will probably be happier if you use Hash Rockets instead.  I.e., 
```
add_reference :physical_servers, :asset_details, :index => true, :type => :bigint
```